blob/blob.go

```diff
@@ -26,53 +25,83 @@ const ProtoIndexWrapperTypeID = "INDX"
 // MaxShareVersion is the maximum value a share version can be. See: [shares.MaxShareVersion].
 const MaxShareVersion = 127
 
+// Blob (stands for binary large object) is a core type that represents data
+// to be submitted to the Celestia network alongside an accompanying namespace
+// and optional signer (for proving the author of the blob)
```

Comment:
uber nit I think we should be careful to not conflate author and signer. A user can author a blob and hand it to a different entity (the blob signer) who signs the PFB tx and submits it on-chain. In other words, the _signer_ can be different from the _author_.

```suggestion
// and optional signer (for proving the signer of the blob)
```

Reply:
As far as Celestia is concerned the signer is always the author but I agree that we should try keep the nomenclature the same throughout (i.e. we use `Signer` in the blob itself)